docker/fullnode/README.md

```diff
@@ -1,83 +1,93 @@
 
 - linux/amd64
 - darwin/amd64 
 - darwin/arm64
 
-# prerequisites
+## Prerequisites
 
-Install docker / docker compose:
+Install Docker / Docker Compose:
 - https://docs.docker.com/get-docker/
 - https://docs.docker.com/compose/install/
 
-# run
+## Running
 
-## fullnode config
+### Fullnode config
 
-Get the latest version of the fullnode config [here](https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml), or:
+Download the latest version of the fullnode config [fullnode-template.yaml](https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml) over the web or by using `curl` or `wget`, for example:
 
-```wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml```
+```shell
+wget https://github.com/MystenLabs/sui/raw/main/crates/sui-config/data/fullnode-template.yaml
+```
+
+TODO: Don't they need to rename this file to `fullnode.yaml` too?
 
-## sui devnet genesis
+### sui devnet genesis
 
-Get the latest version of the Sui DevNet genesis blob [here](https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob), or:
+Get the latest version of the Sui DevNet genesis [genesis.blob](https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob) file over the web or:
 
 ```wget https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob```
 
 
-## start the fullnode
+## Start the fullnode
 
-```docker-compose up```
+To start the fullnode using Docker, run:
 
-# test
+```shell
+docker-compose up
+```
 
-Once the fullnode is up and running, test some of the jsonrpc interfaces.
+## Test
 
-- get the five most recent transactions:
+Once the fullnode is up and running, test some of the JSON-RPC interfaces.
+
+- Get the five most recent transactions:
 
 ```
 curl --location --request POST 'http://127.0.0.1:9000/' \
     --header 'Content-Type: application/json' \
     --data-raw '{ "jsonrpc":"2.0", "id":1, "method":"sui_getRecentTransactions", "params":[5] }'
 ```
 
-- get details about a specific transaction:
+- Get details about a specific transaction:
 
 ```
 curl --location --request POST 'http://127.0.0.1:9000/' \
     --header 'Content-Type: application/json' \
     --data-raw '{ "jsonrpc":"2.0", "id":1, "method":"sui_getTransaction", "params":["$RECENT_TXN_FROM_ABOVE"] }'
 ```
 
-# use your fullnode with explorer 
+## Use your fullnode with Explorer 
 
-To use the DevNet explorer with your fullnode follow these steps:
+To use the Sui Explorer with your fullnode, follow these steps:
 - Open https://explorer.devnet.sui.io
-- Click the green Devnet button in the top right
+- Click the green **Devnet** button in the top right
 - Select 'Custom RPC URL'
 - Set it to http://127.0.0.1:9000
 
-# troubleshoot / tips / documentation
+TODO: Confirm this shouldn't be Local as in the existing fullnode doc.
```

Comment:
At least for me when I attempt to use 'Local' with my fullnode instance it doesn't work. I need to explicitly point the explorer at the 'Custom RPC URL' http://127.0.0.1:9000

I do remember seeing some chatter about the Local default being changed however I am not sure if this has or is happening.. /cc @666lcz / @stella3d ?